scripts/LSF_dynamic_workers.sh

```diff
@@ -32,7 +32,7 @@ hostnamectl set-hostname ${lsfWorkerhostname}
 host_prefix=$(hostname|cut -f1-4 -d -)
 
 # NOTE: On ibm gen2, the default DNS server do not have reverse hostname/IP resolution.
-# 1) put the master server hostname and ip into lsf hosts.
+# 1) put the controller server hostname and ip into lsf hosts.
```

Comment:
We did settle on changing "controller" to "management node" in the next release, if time allows, so this is fine.  In fact, looking at all the uses of "controller" we may want to rethink that further change!  It doesn't match up with what's used externally, but we may be able to live with it within the code.

Reply:
@AugieMena3 We decided that this will be fixed in next release